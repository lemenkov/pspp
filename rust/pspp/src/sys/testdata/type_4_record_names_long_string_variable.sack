# File header.
"$FL2"; s60 "$(#) SPSS DATA FILE PSPP synthetic test file";
2; 2; 1; 0; -1; 100.0; "01 Jan 11"; "20:53:52"; s64 ""; i8 0 *3;

# Long string variable.
2; 9; 0; 0; 0x010900 *2; s8 "STR1";
2; -1; 0; 0; 0; 0; s8 "";

# Value label that names long string variable.
3; 1; s8 "xyzzy"; i8 3; s7 "one"; 4; 1; >>1<<;

# Character encoding record.
7; 20; 1; 12; "windows-1252";

# End of dictionary.
999; 0;
