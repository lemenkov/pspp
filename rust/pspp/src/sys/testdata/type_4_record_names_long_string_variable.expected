At offsets 0xf4...0x114, record types 3 and 4 may not add value labels to one or more long string variables: STR1

╭──────────────────────┬────────────────────────╮
│       Created        │    01-JAN-2011 20:53:52│
├──────────────────────┼────────────────────────┤
│Writer Product        │PSPP synthetic test file│
├──────────────────────┼────────────────────────┤
│       Compression    │SAV                     │
│       Number of Cases│Unknown                 │
╰──────────────────────┴────────────────────────╯

╭─────────┬─╮
│Variables│1│
╰─────────┴─╯

╭────┬────────┬─────┬─────────────────┬─────┬─────┬─────────┬────────────┬────────────┬──────────────╮
│    │Position│Label│Measurement Level│ Role│Width│Alignment│Print Format│Write Format│Missing Values│
├────┼────────┼─────┼─────────────────┼─────┼─────┼─────────┼────────────┼────────────┼──────────────┤
│str1│       1│     │Nominal          │Input│    9│Left     │A9          │A9          │              │
╰────┴────────┴─────┴─────────────────┴─────┴─────┴─────────┴────────────┴────────────┴──────────────╯
